packages/web-components/src/components/va-file-input/va-file-input-upgrader.ts

```diff
@@ -0,0 +1,609 @@
+/* eslint-disable i18next/no-literal-string */
+import { Sanitizer, selectOrMatches } from '../../utils/utils';
+
+const PREFIX = 'usa';
```

Comment:
Can you mention what (if any) changes we needed to make from the original USWDS script in order to get this working for the web component?

https://github.com/uswds/uswds/blob/202448875393d39d663614d2d8f9a12c91fed75e/packages/usa-file-input/src/index.js

Reply:
I didn't keep track, but there are a lot of changes.

Reply:
I was able to create a basic diff here: https://www.diffchecker.com/D8NTH2nR/

Reply:
Thanks!